specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/compute.json

```diff
@@ -7851,7 +7858,7 @@
           "format": "date-time",
           "description": "The UTC timestamp when the operation began."
         },
-        "patches": {
+        "availablePatches": {
```

Comment:
>availablePatches [](start = 9, length = 16)

I compared with [https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json](https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json) and it has the `patches` property in that version. This appears to be a breaking change. Can you explain?

Reply:
The property is renamed in the API since the last release and is in Prod. Here’s the reference to it: 
https://msazure.visualstudio.com/One/_git/Compute-CPlat-Core?path=%2Fsrc%2FCRP%2FDev%2FComputeResourceProvider%2FApiEntityModel%2FVMGuestPatch%2FVMGuestPatchApiEntityModel.cs&version=GBmaster&line=478&lineEnd=479&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents

Reply:
@majastrz assessPatches, the API this property belongs to is in preview mode and I see the instructions are different for previews: https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#preview-versions. Let us know if this change is okay or if we need to do anything else. NOTE: not sure where to add the suffix ‘preview.X’ to indicate this is in preview mode.

Reply:
Change approved by API review board: https://msazure.visualstudio.com/One/_workitems/edit/8875591?src=WorkItemMention&src-action=artifact_link